types/amazon-connect-streams/index.d.ts

```diff
@@ -0,0 +1,688 @@
+// Type definitions for non-npm package Amazon Connect Streams API 1.3
+// Project: amazon-connect-streams
```

Comment:
This needs to be link to the project without that we cannot verify if definitions actually match the structure you are defining here.

Reply:
I have amended the .d.ts Project header to include a link to the amazon-connect-streams project.